### ISSUE TITLE
minor documentation improvement

### ISSUE THREAD

**Issue by hujuice** [gphotosuploader/gphotos-uploader-cli#338]
Just a minor documentation proposal.
It could be helpful mostly after the bug #331.

I'm running `gphotos-uploader-cli` on a headless machine and I cant run straight the auth process when I am required to visit `http://localhost:12345` (where 12345 is a random port number).

Everyone is able to find a workaround, but a proposal in the documentation could save some time.
For example, my workaround is to start a second `ssh` connection with port forwarding:
`ssh -L 12345:localhost:12345 myremotehost`
Now the same port on the real local host is forwarded to the remote host.

Regards,
Sergio

**Comment by pacoorozco** [comment]
Hi Sergio

Thanks for using the `gphotos-uploader-cli` and for reach me. **Where do you suggest to document this hack?** 

I said several times that the `gphotos-uploader-cli` [was not intended to be used in a headless environment](https://github.com/gphotosuploader/gphotos-uploader-cli/issues/320), but I'm seeing a lot of people is using it, so I was wondering where to document the whole headless setup.

**Comment by hujuice** [comment]
Just here: https://gphotosuploader.github.io/gphotos-uploader-cli/#/getting-started?id=authentication
It's ok if it is not intended for headless hosts, but since it works, a tip is an option.

A few weeks ago I changed host: uff, I had to find a way to reach the "localhost". Now there is the authentication issue: uff, manpage again to do the same. Maybe I will use it next month, again the same...
Having it written makes the life better.

Thank you to you, Paco, for this useful software. I'm working as photographer, now, and it's very helpful.
